docs/UserGuide.md

```diff
@@ -372,6 +380,23 @@ Examples:
 - `delete-p 1 1` Deletes the 1st person of the 1st startup.
 - `delete-p 2 3` Deletes the 3rd person of the 2nd startup.
 
+<box type="tip" seamless>
+
+**Tip:** Always click on the startup card after performing person-related operations to ensure you see the updated information.
+</box>
+
+### Clearing all entries : `clear`
+
+Clears all entries from CapitalConnect.
+
```

Comment:
Maybe it would be good to add a tip box warning on the execution of the clear command.